graphql-kotlin-schema-generator/src/main/kotlin/com/expediagroup/graphql/generator/extensions/kPropertyExtensions.kt

```diff
@@ -27,6 +28,7 @@ import kotlin.reflect.KProperty
 internal fun KProperty<*>.isPropertyGraphQLIgnored(parentClass: KClass<*>): Boolean = when {
     this.isGraphQLIgnored() -> true
     getConstructorParameter(parentClass)?.isGraphQLIgnored().isTrue() -> true
+    this.javaGetter?.isGraphQLIgnored().isTrue() -> true
```

Comment:
instead of `javaGetter` why not just `getter`?

Reply:
Yep, I can use that instead and update the extensions